hdfs-scheduler/src/main/java/org/apache/mesos/hdfs/scheduler/HdfsSchedulerModule.java

```diff
@@ -0,0 +1,16 @@
+package org.apache.mesos.hdfs.scheduler;
```

Comment:
I can't tell what this is referring to?   This class is a guice module and doesn't have a constructor.